Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveStorageFolder.cs

```diff
@@ -426,6 +428,81 @@ public OneDriveStorageFolder(IOneDriveClient oneDriveProvider, IItemRequestBuild
         }
 
         /// <summary>
+        /// Download the <see cref="OneDrive.OneDriveStorageFolder"/> in the background
+        /// </summary>
+        /// <param name="destinationFolder">Destination <see cref="StorageFolder"/></param>
+        /// <param name="filter">Filters the response based on a set of criteria.</param>
+        /// <param name="completionGroup">The <see cref="BackgroundTransferCompletionGroup"/> to be used with <see cref="BackgroundDownloader"/></param>
+        /// <param name="successToast">The <see cref="ToastNotification"/> to be shown when download is completed</param>
```

Comment:
You mean the toast? Backgrounddownloader has success/failure toasts option once all the files are finished - I just thought about exposing them, so that all the files will belong to the same 'toast group'.